README.md

```diff
@@ -18,6 +18,28 @@ This project is a template for [React Native](https://reactnative.dev/) that can
 
 NodeJS (version 16 or later) and NPM is required.
 To make sure you have them available on your machine, try running the following command
+## About Us
+
+Welcome to Awesome Library! We're a small team of developers passionate about creating library that make people's lives easier.
+
+### Mission
+
+Our mission is to create the best library for implementing project stucture easily. We believe that everyone can benefit from tools that help them work smarter, not harder. We're committed to building an library that's easy to use, reliable, and affordable for everyone.
+
+### History
+
+My library was founded on Oct 2022 with the goal of making productivity library more accessible to everyone. Since then, we've been working tirelessly to improve the library and add new features.
+
+### Contact Us
+
+If you have any questions or feedback, we'd love to hear from you! You can reach us at hong.hoang@saigontechnology.com.
```

Comment:
I think we should use our STS-ReactNative email

Reply:
Solved